Tella/Scenes/CommonConnectionReport/ViewModels/OutboxMainViewModel.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  OutboxMainViewModel.swift
+//  Tella
+//
+//  Created by gus valbuena on 7/10/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class OutboxMainViewModel<T: Server>: ObservableObject {
```

Comment:
maybe the same here https://github.com/Horizontal-org/Tella-iOS/pull/171#discussion_r1720022555

Reply:
for this one is necessary to add the generic because then we need to access specific properties of TellaServer

```
class OutboxReportVM: OutboxMainViewModel<TellaServer> {    
    var reportRepository = ReportRepository()
    
    var reportIsNotAutoDelete: Bool {
        return !(reportViewModel.server?.autoDelete ?? true)
    }
  ```
  
  Same for the comment below of `ReportViewModel`
  
  Is there a better way to do this?

Reply:
yeah you're right

Reply:
I think it will be unuseful to add `server: GDriveServer?` in  `GDriveOutboxViewModel` and use the server in reportViewModel ?